### ISSUE TITLE
Specific head gear not attaching to head model when loading poses

### ISSUE THREAD

**Issue by bigluobo** [imchillin/Anamnesis#499]
The Endless summer glasses will detach and stay in place when a pose is loaded. Occasionally it spins on its own along the X axis if Unfroze Physics
![20210831023606](https://user-images.githubusercontent.com/6994715/131453995-2bbd6c6c-b1d3-424e-a1ad-8491d3a27dee.png)

**Comment by Mothism** [comment]
This is a common occurrence (not necessarily an issue) with gear that can be /visor'd as the placement of the model is either adjusted or forced inside of the character when you use the visor action. Gear with this property are on their own bone (typically Headgear), which isn't usually accounted for in released poses. As for the spinning, this usually happens when a piece of headgear is adjusted rather than removed, such as the Endless Summer Glasses moving onto the head—this happens when you adjust physics in the CMTool as well.

tl;dr, not an issue, just a bone that's usually not accounted for.

**Comment by bigluobo** [comment]
Thank you that explains alot!
